keras/src/saving/file_editor_test.py

```diff
@@ -98,14 +98,16 @@ def test_scalar_weight(self):
         model.add(keras.layers.Dense(1, activation="sigmoid", name="my_dense"))
         model.compile(optimizer="adam", loss="mse", metrics=["mae"])
         model.fit(np.array([[1]]), np.array([[1]]), verbose=0)
-        model.save("model.keras")
-        model.save_weights("model.weights.h5")
+        model_fpath = os.path.join(self.get_temp_dir(), "model.keras")
+        weights_fpath = os.path.join(self.get_temp_dir(), "model.weights.h5")
```

Comment:
Good improvement using `self.get_temp_dir()` for test file paths. This prevents potential file conflicts and follows test best practices for file management.